python_src/src/lamp_py/migrations/versions/performance_manager_prod/002_1b53fd278b10_fix_trip_id_length.py

```diff
@@ -0,0 +1,93 @@
+"""fix trip id length
+
+Revision ID: 1b53fd278b10
+Revises: 5d9a7ee21ae5
+Create Date: 2023-11-27 16:25:42.657967
+
+Details
+* upgrade -> change "trip_id" field from length 128 to 512
+* upgrade -> add static service id lookup table
+
+* downgrade -> change "trip_id" field from length 512 to 128
+* downgrade -> drop static service id lookup table 
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1b53fd278b10'
+down_revision = '5d9a7ee21ae5'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('static_service_id_lookup',
+    sa.Column('dummy_pk', sa.Integer(), nullable=False),
+    sa.Column('route_id', sa.String(length=60), nullable=True),
+    sa.Column('service_id', sa.String(length=60), nullable=True),
+    sa.Column('service_date', sa.Integer(), nullable=True),
+    sa.Column('static_version_key', sa.Integer(), nullable=True),
+    sa.PrimaryKeyConstraint('dummy_pk')
+    )
```

Comment:
this somehow wasn't in the original migration script. it got picked up when i ran the alembic generation.